examples/tx-example.py

```diff
@@ -200,5 +215,14 @@ def get_userone_details():
 get_coin_info()
 get_account_assets()
 get_userone_details()
+# set timestamp to correct value
+# for more protobuf timestamp api info see:
+# https://googleapis.dev/python/protobuf/latest/google/protobuf/timestamp_pb2.html
+first_time=Timestamp()
+first_time.FromMilliseconds(1626249925185)
+last_time=Timestamp()
+last_time.FromMilliseconds(1626250078804)
+print(first_time, last_time)
```

Comment:
I suggest to make logging more verbose, not just printing values. If it is timestamp I suggest to print in more human-readable form, e.g. use function https://www.tutorialspoint.com/python/time_strftime.htm and print as `datetime` object

Reply:
I think I will add neccessary comments and resing from printing values